docs/reference/content/driver-scala/getting-started/quick-start-case-class.md

```diff
@@ -11,7 +11,7 @@ title = "Quick Start - Case Classes"
 # Quick Start with case classes
 
 The following code snippets come from the `QuickTourCaseClass.scala` example code that can be found with the 
-[driver source]({{< srcref "examples/src/test/scala/tour/QuickStartCaseClass.scala">}}).
+[driver source]({{< srcref "examples/src/test/scala/tour/QuickStartCaseClass.scala" >}}).
```

Comment:
Is the space required?

Reply:
This was just part of the normalization done via cleaning up and find/replace